src/main/java/dinkplugin/message/Field.java

```diff
@@ -20,6 +21,10 @@ public class Field {
     @Nullable Boolean inline;
 
     public Field(String name, String value) {
-        this(name, value, null);
+        this(name, value, true);
+    }
+
+    public static String formatBlock(String codeBlockLanguage, String content) {
```

Comment:
I'm also open to moving this method to Utils even though it's only used for field value formatting

Reply:
I'm fine with this staying here until we find a use for it outside of Fields